examples/analysis/SUopt.flix

```diff
@@ -117,3 +117,11 @@ def top(e: Int): SULattice = SULattice.Top;
 Kill(l,f(b)) :- Store(l,p,q), Pt(p,b).
 Kill(l, top(42)) :- Phi(l,_,_).
 
+// uncomment to introduce an error delta debugging
```

Comment:
If I'm understanding correctly, the reason for inserting an error into SUopt is because SU is a very large benchmark with a very large input, so it's a great demo for delta debugging. But it's also a real, working example, and it's messy to have commented out code and extra stuff lying around in it.

I would suggest putting the error in a separate file, and then providing both files (plus input facts) as input to Flix.